<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IncidenceAttachmentEditor</class>
 <widget class="QWidget" name="IncidenceAttachmentEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>427</width>
    <height>111</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0" rowspan="2" colspan="3">
    <widget class="QWidget" name="mAttachmentViewPlaceHolder" native="true"/>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="mAddButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string extracomment="@info:tooltip">Add an attachment</string>
     </property>
     <property name="whatsThis">
      <string extracomment="@info:whatsthis">Shows a dialog used to select an attachment to add to this event or to-do as link or as inline data.</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="mRemoveButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string extracomment="@info:tooltip">Remove the selected attachment</string>
     </property>
     <property name="whatsThis">
      <string extracomment="@info:whatsthis">Removes the attachment selected in the list above from this event or to-do.</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
